lib/Ohmbrewer_Screen.h

```diff
@@ -213,6 +213,11 @@ namespace Ohmbrewer {
              */
             Menu* _homeMenu;
 
+           /**
```

Comment:
The same way you're doing it, except mainMenu would be a local variable instead of an instance variable. It should work because it's a pointer.

This actually highlights a bug though! We don't have a destructor for this class (we just rely on the parent class's destructor), which means that if somehow the screen was deleted, the touchscreen and menu instance variables would turn into a memory leak (the other variables are managed separately). We've never noticed before because we never delete the screen. We should still provide that though. You can either take a whack at it here, or open an issue that references this since it's kind of an increase in scope.